phpstan.neon

```diff
@@ -0,0 +1,18 @@
+parameters:
+  level: 3
+  paths:
+    - ./src
+  ignoreErrors:
+    - '#Call to [a-zA-Z0-9\\_:@\(\).]*#'
```

Comment:
E.g:

1. `Call to static method service() on an unknown class Drupal.`
2. `Call to static method getContainer() on an unknown class Drupal.`
3. `Call to an undefined method Acquia\Console\ContentHub\Tests\Drupal\Command\ContentHubAuditCheckUuidTest::assertNotContains().`